ONSdigital/sdx-transform-cs
# Use sequence id within scan id increment

**Changes**

- Added behaviour to increment scan ids using the sequence number, removing need for maintaining a separate scan id as well as sequence number.
- Unit tests have been added to check common-software zip contents

**How to test**

Check that unit tests pass.

To manually test use a response payload (example found in views/test_views.py) to POST against 127.0.0.1:5000/common-software, this should return a zip where scan id's increment on default sequence number (1000), so resulting in an id S010000001.jpg. Appending the sequence id to the endpoint should return a zip with a scan id using that sequence id. eg. 127.0.0.1:5000/common-software/5678 returns S056780001.jpg.

**Who can test**

Anyone but @iwootten
